applications/gui/modules/text_input.c

```diff
@@ -259,172 +259,171 @@ static void text_input_view_draw_callback(Canvas* canvas, void* _model) {
     }
 }
 
-static void text_input_handle_up(TextInput* text_input) {
-    with_view_model(
-        text_input->view, (TextInputModel * model) {
-            if(model->selected_row > 0) {
-                model->selected_row--;
-                if(model->selected_column > get_row_size(model->selected_row) - 6) {
-                    model->selected_column = model->selected_column + 1;
-                }
-            }
-            return true;
-        });
+static void text_input_handle_up(TextInput* text_input, TextInputModel* model) {
```

Comment:
Here and below (in all `text_input_handle_` functions, except `handle_ok`), TextInput* is no longer needed.